src/Chartjs/DataSets/DoughnutAndPie.elm

```diff
@@ -1,44 +1,124 @@
-module Chartjs.DataSets.DoughnutAndPie exposing (DataSet, defaultBarFromLabel)
+module Chartjs.DataSets.DoughnutAndPie exposing
+    ( DataSet, defaultPieFromLabel, defaultPieFromData
+    , setData, setBackgroundColor, setBorderAlign, setBorderColor, setBorderWidth
+    , setHoverBackgroundColor, setHoverBorderColor, setHoverBorderWidth
+    )
+
+{-| Pie and doughnut charts are divided into segments, where the arc of each segment shows the proportion of each data.
+
+In Chart.js, these two charts are essentially the same. The only different value is the cutoutPercentage.
+This dataset class will work for either pie or doughtnut charts interchangably
+
+@docs DataSet, defaultPieFromLabel, defaultPieFromData
+@docs setData, setBackgroundColor, setBorderAlign, setBorderColor, setBorderWidth
+@docs setHoverBackgroundColor, setHoverBorderColor, setHoverBorderWidth
+
+-}
 
 import Chartjs.Common as Common
 import Color exposing (Color)
 
 
+{-| For further information on these properties, see <https://www.chartjs.org/docs/latest/charts/doughnut.html>
+
+You should not use the dataset type directly
+Instead use the updater pipeline functions:
+
+    defaultBarFromLabel "Example"
+        |> setBackgroundColor (Common.All Color.red)
+        |> setBorderColor (Common.All Color.white)
+
+-}
 type alias DataSet =
     { label : String
-    , -- The label for the dataset which appears in the legend and tooltips.
-      data : List Float
+    , data : List Float
     , backgroundColor : Maybe (Common.PointProperty Color)
-    , -- The fill color of the arcs in the dataset.
-      borderColor : Maybe (Common.PointProperty Color)
-    , -- The border color of the arcs in the dataset.
-      borderWidth : Maybe (Common.PointProperty Float)
-    , -- The border width of the arcs in the dataset.
-      hoverBackgroundColor : Maybe (Common.PointProperty Color)
-    , -- The fill colour of the arcs when hovered.
-      hoverBorderColor : Maybe (Common.PointProperty Color)
-    , -- The stroke colour of the arcs when hovered.
-      hoverBorderWidth : Maybe (Common.PointProperty Float)
-    , -- The stroke width of the arcs when hovered.
-      cutoutPercentage : Maybe Int
```

Comment:
Don't forget to move this into the options! They're pretty useful parameters to have